### PR TITLE
Revert "[CPT-2535] Add property to place accordion icon on the left"

### PR DESCRIPTION
**Pull request by angelinastavniiciuc**
Reverts toptal/picasso#4522

### CODE REVIEW

**Comment by sashuk** [comment]
@angelinastavniiciuc Can you please add changeset?

**Comment by angelinastavniiciuc** [comment]
> Member

@sashuk yup, on it, just not sure which bump should it be major or minor ? and whether I should include all changed packages or only accordion ?

**Review by TomasSlama — APPROVED** [review]

**Comment by sashuk** [comment]
@angelinastavniiciuc

> which bump should it be major or minor ?

This should be a major one, as new public API of Accordion is going to be incompatible with the old one

> include all changed packages or only accordion ?

Let's include all packages that changeset will mark as "updated"

File: .changeset/violet-frogs-bake.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+'@toptal/picasso-rich-text-editor': major
```
Comment (sashuk) [inline .changeset/violet-frogs-bake.md:2]: Just to confirm, affected packages were generated by changeset?

Comment (angelinastavniiciuc) [reply .changeset/violet-frogs-bake.md:2]: yep, after running `yarn changeset` this was resulting list of changed packages
image

Comment (sashuk) [reply .changeset/violet-frogs-bake.md:2]: @angelinastavniiciuc Got it, thanks!

**Review by sashuk — APPROVED** [review]

**Comment by angelinastavniiciuc** [comment]
@sashuk can you please help with failing Happo check ? (it is because the added visual test was removed now)

**Comment by sashuk** [comment]
@angelinastavniiciuc Done!

**Pull request merged**